### PR TITLE
Add the command line to each task log. Closes #3941

### LINKED ISSUE #3941: The `cmd` field of WesLog is always None
It's not supposed to be

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
Request for a centaur test to validate that this is doing what you expect

**Comment by geoffjentry** [comment]
@cjllanwarne  If you're offering to embed wes2cromwell into centaur, sure

**Comment by cjllanwarne** [comment]
It looks like it should appear in the basic Cromwell metadata too?

**Comment by geoffjentry** [comment]
Good point

File: centaur/src/main/scala/centaur/test/metadata/WorkflowMetadata.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ case class WorkflowMetadata(value: Map[String, JsValue]) extends AnyVal {
     // If the test fails in initialization there wouldn't be workflow root metadata, and if that's the expectation
     // then that's ok.
     val workflowRoot = actual.value.get("workflowRoot").collectFirst { case JsString(r) => r } getOrElse "No Workflow Root"
-    val missingErrors = value.keySet.diff(actual.value.keySet) map { k => s"Missing key: $k" }
+    val missingErrors = value.keySet.diff(actual.value.keySet) map { k => s"Missing key in worflow $workflowID: $k" }
```
Comment (mcovarr) [inline centaur/src/main/scala/centaur/test/metadata/WorkflowMetadata.scala]: worflow son of moghlow

Comment (geoffjentry) [reply centaur/src/main/scala/centaur/test/metadata/WorkflowMetadata.scala]: you butt out of my mystery solving

**Pull request merged**